test/dummy/app/models/widget.rb

```diff
@@ -7,16 +7,13 @@ class Widget < ActiveRecord::Base
 
   self.doctype.mapping[:properties].update(
     'name' => {
-      type: 'string', index: 'not_analyzed',
-      fields: {
-        analyzed: {type: 'string', index: 'analyzed'}
-      }
+      type: 'text',
```

Comment:
This seemed to be the closest equivalent mapping I could determine for this property. I couldn't find any documentation (for previous or current versions) on the `field: analyzed: {...}` mapping.

Reply:
text is good.

Reply:
Except it looks like we lose tests for `name` (keyword) vs `name.analyzed` (text), right?

Reply:
Possibly. I updated the mapping, probably more accurate now.